### PR TITLE
variants/ns5x_adl: add v1.7.2 relnotes

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

File: docs/variants/novacustom_ns5x_adl/releases.md [hunk 1]
```diff
@@ -13,6 +13,80 @@ For details about our release process please read
 
 </center>
 
+## v1.7.2 - 2023-12-22
+
+Test results for this release can be found
+[here](https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=799846462).
```
Comment (BeataZdunczyk) [inline docs/variants/novacustom_ns5x_adl/releases.md:19]: @mkopec please fix spreadsheet link to: https://docs.google.com/spreadsheets/d/1LOXY9HCu-fMitkYwX08iLsQdSNenzyU0LnMdVbZB5Do/edit#gid=271422464

File: docs/variants/novacustom_ns5x_adl/releases.md [hunk 2]
```diff
@@ -13,6 +13,81 @@ For details about our release process please read
 
 </center>
 
+## v1.7.2 - 2023-12-22
```
Comment (BeataZdunczyk) [inline docs/variants/novacustom_ns5x_adl/releases.md:16]: We need to adjust the release date (likely to 2023-12-28). However, we will correct it before merging, so there's no need to fix that today.

**Review by miczyg1 — CHANGES_REQUESTED** [review]
The Openness Score report in MD is missing

File: docs/variants/novacustom_ns5x_adl/releases.md (hunk 2 above)
Comment (mkopec) [reply docs/variants/novacustom_ns5x_adl/releases.md]: Release date adjusted to today

**Review by macpijan — APPROVED** [review]

**Pull request merged**